### PR TITLE
austin.intro.1.russ.savage

### PR DESCRIPTION
**Pull request by RX7RUSS**

### CODE REVIEW

File: 01Lesson/index.html [hunk 1]
```diff
@@ -1 +1,10 @@
-Your first website
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>First Class Change</title>
+</head>
+<body>
+  <p> paragraph 1 </P>
```
Comment (ivome) [inline 01Lesson/index.html]: Always try to use lower case tags.

File: 02Lesson/01Homework/index.html [hunk 2]
```diff
@@ -1,8 +1,32 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title></title>
+  <meta charset="utf-8">
+  <title>Savage Bio</title>
 </head>
 <body>
+  <img style="height:20%; width: 20%" src = "https://scontent-mia1-1.xx.fbcdn.net/v/t1.0-9/10653714_10204131189073222_5438750846180900030_n.jpg?oh=78f887afeb78a1dfb85465779e5484f9&oe=58A3F75D" alt="Russ savage"/>
```
Comment (ivome) [inline 02Lesson/01Homework/index.html]: No spaces around `=` sign for attributes.

File: 02Lesson/01Homework/index.html [hunk 3]
```diff
@@ -1,8 +1,32 @@
...
+  <h1>Russ Savage</h1>
+  <hr>
+  <p>
+  I moved to Austin 4 years ago and love it here. The summers can be brutal but after living in extremely cold places like South Dakota, I can tolerate heat in the summer in order to have 50 degree days in the winter.
```
Comment (ivome) [inline 02Lesson/01Homework/index.html]: Indentation: Content of `p` element should be indented.

File: 02Lesson/01Homework/index.html [hunk 4]
```diff
@@ -1,8 +1,32 @@
...
+  </p>
+  <h2>Hobbies and Interests</h2>
+  <p>My primary hobby right now is racing RC cars. I race both on and off road. I run 2 classes in both types of racing</p>
+  <img style="height:20%; width: 20%" src="https://scontent-mia1-1.xx.fbcdn.net/v/t1.0-9/13439232_10208523386995425_2067274129786750026_n.jpg?oh=a8b35f8e7d963d4417b694db5059c79e&oe=585FCFCB" alt="RC Cars">
+    <ul>
+      <li><h3>On Road Classes</h3>
```
Comment (ivome) [inline 02Lesson/01Homework/index.html]: I would put tags of block elements on their own line:

```
<li>
  <h3>
     On Road Classes
  </h3>
  <ol>
     ...
  </ol>
</li>
```

That way you see immediately that indentation for `<ol>` tag is on the wrong level which leads to the missing closing ´</li>` tag a few lines further down.

File: 02Lesson/01Homework/index.html [hunk 5]
```diff
@@ -1,8 +1,32 @@
...
+      <ol>
+        <li>USGT</li>
+        <li>17.5 Stock</li>
+      </ol>
```
Comment (ivome) [inline 02Lesson/01Homework/index.html:25]: Missing closing `li` tag.

File: 02Lesson/01Project/index.html [hunk 6]
```diff
@@ -1,9 +1,33 @@
 <!DOCTYPE html>
 <html>
-<head>
-  <title></title>
-</head>
-<body>
-
-</body>
+    <head>
```
Comment (ivome) [inline 02Lesson/01Project/index.html:3]: Indentation: Always use 2 space instead of 4.

File: 02Lesson/01Project/index.html [hunk 7]
```diff
@@ -1,9 +1,33 @@
...
+          <meta charset="utf-8">
```
Comment (ivome) [inline 02Lesson/01Project/index.html]: Indentation: All child elements of `head` element should be on the same level.

File: 02Lesson/01Project/index.html [hunk 8]
```diff
@@ -1,9 +1,33 @@
...
+      <title>
+        Allisons Custom Dreamcatchers
+      </title>
+    </head>
+    <body>
+      <img src="http://cdn1.dreamcatchercomi.netdna-cdn.com/images/A/dc775-purple-soul-connection-with-swarovski-crystal-dreamcatcher.jpg" alt="Dreamcatcher image">
+        <h1>
+          Allison's Custom Dreamcatchers
+        </h1>
+          <p>
```
Comment (ivome) [inline 02Lesson/01Project/index.html]: Indentation: After closing `h1` tag, next element needs to be on the same level.

File: 02Lesson/01Project/index.html [hunk 9]
```diff
@@ -1,9 +1,33 @@
...
+            <h3>
+            Services We Provide
```
Comment (ivome) [inline 02Lesson/01Project/index.html]: Indentation: Content should be indented 2 spaces.

File: 02Lesson/01Project/index.html [hunk 10]
```diff
@@ -1,9 +1,33 @@
...
+                <li>
+                  Hand Made by Allison
+                </li>
+                <li>
+                  Designed specially for you
+                </li>
+              </ul>
+          <p>
+            Contact us ia email at: <a href="allison@dreamcatchers.com">Email Us</a>
+          </p>
```
Comment (ivome) [inline 02Lesson/01Project/index.html]: Inconsistent indentation, always use 2 spaces everywhere 😄

File: 03Lesson/01Homework/index.html [hunk 11]
```diff
@@ -1,10 +1,23 @@
...
+      <div>
+      <img src="img/berries.jpg" alt="berries" />
+      </div>
+    </td>
+    <td>
+      <div class="berries">
+        Berries
+      </div>
+    </td>
+</table>
```
Comment (ivome) [inline 03Lesson/01Homework/index.html]: This is probably work in progress? Check the indentation: `img` inside of `div` needs to be indented. Missing closing `tr` tag.

File: 03Lesson/01Project/css/style.css [hunk 12]
```diff
@@ -1 +1,51 @@
...
+  margin: auto;
+}
+
+header {
+  background-color: #0071c5;
+  padding: 10px;
+  color: white
+}
+
+header-#h1 {
```
Comment (ivome) [inline 03Lesson/01Project/css/style.css]: Interesting selector, is that `-` there by accident? 😄 Doesn't look like valid CSS

File: 03Lesson/01Project/css/style.css [hunk 13]
```diff
@@ -1 +1,51 @@
...
+  font-size: 24px;
+  font-weight: 200;
+}
+
+main > p {
+  font-size: 16px;
+  color: #555;
+}
+
+body html aside {
```
Comment (ivome) [inline 03Lesson/01Project/css/style.css]: This selector does not make sense. There can never be a `html` element inside of a `body` element in valid HTML markup.

File: 03Lesson/01Project/index.html [hunk 14]
```diff
@@ -1,9 +1,64 @@
...
+    </h1>
+    <nav class="nav-link">
+      <span>i7 Procressor</span>
+      <span>Solid-State Drives</span>
+    </nav>
+  </header>
+  <main>
+    <section>
+      <h2>Intel® Core™ i7</h2>
+      <p>The 6th Generation Intel® Core™ i7 processors deliver a new class of computing with a host of new features to power the next generation of desktops, laptops, and 2 in 1 PCs. Expect lighting fast speeds and peak performance through even the toughest of tasks and games.</p>
```
Comment (ivome) [inline 03Lesson/01Project/index.html]: I would put all block element tags on their own line and indent the content and child elements.

**Review by ivome — COMMENTED** [review]
Great job Russ! I only saw minor issues with indentation ... Always check the W3C validator errors when saving a file in atom and try to make the test suite run. That should not throw any errors once you fix the indentation issues.

**Comment by RX7RUSS** [comment]
Thanks for all the feedback Ivo! Yes the berries project is definitely in progress. I’m having trouble with that one but I’m trying to get in the habit of making lots of commits and syncing even when I know things are wrong.
The CSS for the Intel project is quite messed up I know. I tried to combine the next project which had a CSS file we were supposed to fix. I’m a little lost there as well. I realize some of those selectors don’t make sense. I didn’t realize my indentation was so bad at the beginning. I have turned on the lines in Atom now so hopefully I have less of those issues moving forward. Thanks for the vote of confidence. I feel like I’m a bit behind now. I’ll just keep at it.
-Russ

**Pull request closed without merging**